pageserver/src/tenant/mgr.rs

```diff
@@ -1016,6 +1016,74 @@ impl TenantManager {
 
         Ok(())
     }
+
+    /// Resetting a tenant is equivalent to detaching it, then attaching it again with the same
+    /// LocationConf that was last used to attach it.  Optionally, the local file cache may be
+    /// dropped before re-attaching.
```

Comment:
Mmm, I think it's a sufficiently generic term that we can use it too.  We have lots of services, more than one of them will have local caches of files.

I took the hit on calling our standby mode "secondary" because we'll sometimes be talking about both things in the same breath in the context of HA, but in general we can't entirely avoid overlapping with terms that postgres folks use.